site/specs-temp/numbers.json

```
@@ -1701,7 +1701,8 @@
     "/accounts/{accountId}/bulkPortins": {
       "get": {
         "tags": [
-          "/accounts"
+          "/accounts",
+          "Porting"
```

```suggestion
          "Porting"
```
I think we want to get rid of `/accounts` in the tags 

unfortunately the docs renderer keys off of the tag and will duplicate this endpoint in the docs